src/datahike/writing.cljc

```diff
@@ -0,0 +1,239 @@
+(ns datahike.writing
```

Comment:
Where do I find tests for this namespace?

Reply:
This namespace is just factored out from the prior `connection` namespace. It is tested with all the existing tests, I just tried to factorize the code base a bit better as `connection` was conflating too many things.